ui/raidboss/data/07-dt/raid/r2s.ts

```diff
@@ -299,20 +353,69 @@ const triggerSet: TriggerSet<Data> = {
           cn: '分摊',
           ko: '쉐어',
         },
-        unknown: Outputs.unknown,
+        combo: {
+          en: '${next} + ${mech}',
+        },
       },
     },
     {
-      id: 'R2S Honey Beeline',
+      id: 'R2S Tempting Twist Initial',
       type: 'StartsUsing',
-      netRegex: { id: ['9186', '9B0C'], source: 'Honey B. Lovely', capture: false },
-      response: Responses.goSides(),
```

Comment:
I think what happened is that I had a merge conflict with  something on my tree that made me accidentally keep the old implementation in.